githubchristophersummitt-warren.md

```diff
@@ -0,0 +1,7 @@
+## Warren, Ohio
+
+### Great places to eat
```

Comment:
It is about 45 minutes east of Akron and 1 hour south of Cleveland.  The best place is the Quicken Loans Arena - home of the Cleveland Cavaliers - NBA Champions!